src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/config.jelly

```diff
@@ -30,7 +34,11 @@
         <c:select/>
     </f:entry>
 
-    <f:entry field="aid" title="Analysis ID Override">
+    <f:invisibleEntry field="aid" title="Analysis ID Override" >
```

Comment:
Is this meant to be used to retrieve a specific analysis instead of simply the most recent? Because "override" makes it sound like it is setting the analysis ID for an analysis.

Maybe "Specific Analysis ID" or similar

Reply:
I suggest we just call this "Analysis ID".  Adding "specific" to it might add a new concept that might create more confusion.  The field is supposed to be hidden, so I think keeping it brief is probably the best action.

Reply:
ok